src/Tribe/Aggregator/Service.php

```diff
@@ -543,13 +541,11 @@ public function get_image( $image_id, $record ) {
 		 * @since 4.6.18
 		 *
 		 * @param  array  $data      Which Arguments
-		 * @param  strng  $image_id  Image ID
+		 * @param  string  $image_id  Image ID
```

Comment:
Don't know if it's in scope for this piece of work, but, since we're fixing this docblock: shouldn't we have three `@param` tags? One for the array being filtered, one for `$record` and one for `$image_id`?

Reply:
Let me do the update as we are already on this one :)

Reply:
Since we're doing this:
```suggestion
		 * @param string	      $image_id  The image post ID.
		 * @param array<string,mixed> $data      The image data.
```